test/DotNetWorkerTests/DefaultModelBindingFeatureTests.cs

```diff
@@ -61,6 +61,47 @@ public async void BindFunctionInputAsync_Populates_ParametersUsingConverters()
             Assert.Equal("0ab4800e-1308-4e9f-be5f-4372717e68eb", guid.ToString());
         }
 
+        [Fact]
+        public async void BindFunctionInputAsync_IgnoreCancellationToken()
```

Comment:
We could also consider ensuring the Cancellation token parameter is populated as expected.

```csharp
// Arrange
var parameters = new List<FunctionParameter>()
{
    new("myQueueItem",typeof(Book)),
    new ("myGuid", typeof(Guid)),
    new ("token", typeof(CancellationToken)),
};
...
// Assert
var cancellationToken = TestUtility.AssertIsTypeAndConvert<CancellationToken>(parameterValuesArray[2]);
Assert.True(cancellationToken.IsCancellationRequested);
```

Reply:
Updated the unit test!